src/System.Private.ServiceModel/tools/SelfHostedWcfService/Program.cs

```diff
@@ -274,6 +266,15 @@ static void Main()
             WebSocketHttpsDuplexTextBufferedTestServiceHost webSocketHttpsDuplexTextBufferedTestServiceHostServiceHost = new WebSocketHttpsDuplexTextBufferedTestServiceHost(typeof(WcfService.WSDuplexService), webSocketHttpsDuplexTextBufferedTestServiceHostbaseAddress);
             webSocketHttpsDuplexTextBufferedTestServiceHostServiceHost.Open();
 
+            //Start the crlUrl service last as the client use it to ensure all services have been started
+            Uri crlUrl = new Uri(string.Format("http://localhost/CrlService.svc", httpPort));
+            WebServiceHost host = new WebServiceHost(typeof(CrlService), crlUrl);
+            WebHttpBinding binding = new WebHttpBinding();
+            host.AddServiceEndpoint(typeof(ICrlService), binding, "");
+            ServiceDebugBehavior stp = host.Description.Behaviors.Find<ServiceDebugBehavior>();
+            stp.HttpHelpPageEnabled = false;
+            host.Open();
+
             Console.WriteLine("All service hosts have started.");
             Console.WriteLine("Press <ENTER> to terminate the self service Host.");
             Console.ReadLine();
```

Comment:
Nit: in use, we found requiring only ENTER made it too easy to kill the Bridge.exe window by mistake.   As long as you are touching this file, I would prefer that the user must type "exit" or something like that.  We found this out the hard way before, so I would like to learn from that.